bugsnag/django/__init__.py

```diff
@@ -45,14 +46,24 @@ def add_django_request_to_notification(notification):
 
     if getattr(request, "session", None):
         notification.add_tab("session", dict(request.session))
-    notification.add_tab("request", {
+    request_tab = {
         'method': request.method,
         'path': request.path,
         'encoding': request.encoding,
         'GET': dict(request.GET),
         'POST': dict(request.POST),
         'url': request.build_absolute_uri(),
-    })
+    }
+    try:
+        aj = 'application/json'
+        if request.META.get('CONTENT_TYPE', '').lower().startswith(aj):
+            if request_tab["method"] == "POST":
+                body = request.body.decode('utf-8')
```

Comment:
In the case  that `body` can't be decoded as UTF-8, this could instead use a `replace` strategy to handle the errors and still include as much as possible in the event on  the dashboard:

```suggestion
                body = request.body.decode('utf-8', 'replace')
```

Reply:
Thanks @kattrali, fixed in 5db02996a3bb847eddda4c58d895f29ec3a289a4